omod/src/main/java/org/openmrs/module/laboratoryapp/fragment/controller/ResultFragmentController.java

```diff
@@ -73,7 +72,49 @@ public SimpleObject saveResult(
 		test.setEncounter(encounter);
 		test = ls.saveLaboratoryTest(test);
 		ls.completeTest(test);
-		
+
+
+		//send patient back to the OPD Queue
+		Patient patient = encounter.getPatient();
+		PatientQueueService queueService = Context.getService(PatientQueueService.class);
+		Concept referralConcept = Context.getConceptService().getConcept(2548);
```

Comment:
@franqq hunting this concept id down will be hell. Is it captured as a constant elsewhere, if not please do so maybe as a class field constant